### PR TITLE
Fix example in readme

### PR DESCRIPTION
**Pull request by millette**
Fastify wasn't instantiated.

### CODE REVIEW

**Comment by millette** [comment]
Also added ```fastify-formbody``` plugin for completeness sake.

File: README.md [hunk 1]
```diff
@@ -19,8 +19,9 @@ npm i fastify-leveldb --save
 Add it to you project with `register` and you are done!  
 You can access LevelDB via `fastify.level`.
 ```js
-const fastify = require('fastify')
+const fastify = require('fastify')()
 
+fastify.register(require('fastify-formbody'))
```
Comment (Eomm) [inline README.md]: this should be necessary only if we send a url encoded content type (with application/json it is not necessary).

Maybe to complete the example could you add a `curl` command to let the user try immediately?

Comment (mcollina) [reply README.md]: +1 for adding a curl snippet instead. However this should be covered by some other fastify docs somewhere else.

Comment (millette) [reply README.md]: What is the next step? Add a curl example and remove formbody dep? Or simply remove formbody and add curl examples elsewhere? If so, where's the best place to document this?

Comment (Eomm) [reply README.md]: > Or simply remove formbody and add curl examples elsewhere?

I'm +1 on this.
The curl example could be simply at the bottom of the example code.

**Comment by millette** [comment]
The first thing that bothered me was that fastify wasn't getting instantiated. Since there's no argument on that aspect, I fixed it in #31 while I was at it.

We can forget about this pull requet (formbody), and I can simply add a curl example in #31 and call it a day :-)

Reopen if you disagree.

**Pull request closed without merging**